gymnasium/dev_wrappers/lambda_action.py

```diff
@@ -0,0 +1,181 @@
+        >>> from gymnasium.wrappers import ClipActionsV0
+        >>> env = gymnasium.make("BipedalWalker-v3")
+        >>> env.action_space
+        Box(-1.0, 1.0, (4,), float32)
+        >>> env = ClipActionsV0(env, (-0.5, 0.5))
+        >>> env.action_space
+        Box(-0.5, 0.5, (4,), float32)
+
+    Clip with only a lower or upper bound:
+        >>> env = gymnasium.make('CarRacing-v1')
+        >>> env.action_space
+        Box([-1.  0.  0.], 1.0, (3,), float32)
+        >>> env = ClipActionsV0(env, (None, 0.5))
+        >>> env.action_space
+        Box([-1.  0.  0.], 0.5, (3,), float32)
+
+    Composite action space example:
+        >>> env = ExampleEnv(action_space=Dict(body=Dict(head=Box(0.0, 10.0, (1,))), left_arm=Discrete(4), right_arm=Box(0.0, 5.0, (1,))))
+        >>> env.action_space
+        Dict(body: Dict(head: Box(0.0, 10.0, (1,), float32)), left_arm: Discrete(4), right_arm: Box(0.0, 5.0, (1,), float32))
+        >>> args = {"right_arm": (0, 2), "body": {"head": (0, 3)}}
+        >>> env = ClipActionsV0(env, args)
+        >>> env.action_space
+        Dict(body: Dict(head: Box(0.0, 3.0, (1,), float32)), left_arm: Discrete(4), right_arm: Box(0.0, 2.0, (1,), float32))
+    """
+
+    def __init__(
+        self, env: gymnasium.Env, args: FuncArgType[typing.Tuple[float, float]]
+    ):
+        """Constructor for the clip action wrapper.
+
+        Args:
+            env: The environment to wrap
+            args: The arguments for clipping the action space
+        """
+        action_space = self._transform_space(env, args)
+
+        super().__init__(
+            env, lambda action, args: jp.clip(action, *args), args, action_space
+        )
+
+
+class ScaleActionsV0(LambdaActionV0):
+    """A wrapper that scales actions passed to :meth:`step` with a scale factor.
+
+    Basic Example:
+        >>> import gymnasium
+        >>> from gymnasium.wrappers import ScaleActionsV0
+        >>> env = gymnasium.make('BipedalWalker-v3')
+        >>> env.action_space
+        Box(-1.0, 1.0, (4,), float32)
+        >>> env = ScaleActionsV0(env, (-0.5, 0.5))
+        >>> env.action_space
+        Box(-0.5, 0.5, (4,), float32)
+
+    Composite action space example:
+        >>> env = ExampleEnv(
+        ...    action_space=Dict(left_arm=Box(-2, 2, (1,)), right_arm=Box(-2, 2, (1,))
+        ... )
+        >>> env = ScaleActionsV0(env, {"left_arm": (-1,1), "right_arm": (-1,1)})
+        >>> env.action_space
+        Dict(left_arm: Box(-1, 1, (1,), float32), right_arm: Box(-1, 1, (1,), float32))
+    """
+
+    def __init__(
+        self, env: gymnasium.Env, args: FuncArgType[typing.Tuple[float, float]]
+    ):
+        """Constructor for the scale action wrapper.
+
+        Args:
+            env: The environment to wrap
+            args: The arguments for scaling the actions
+        """
+        action_space = self._transform_space(env, args)
+        args = self._make_scale_args(env, args)
+
+        def scale(action, args):
```

Comment:
I don't think we need to define the function inside of the method, it's better to keep it outside if it's even necessary